src/components/AuthErrorsContainer/AuthErrorsContainer.js

```diff
@@ -21,7 +21,12 @@ export default class AuthErrorsContainer extends Component {
     const values = parameters.reduce((res, { key, value }) => ({ ...res, [key]: value }), {});
 
     return (
-      <li key={`${code}-${type}`}>
+      <li
+        key={`${code}-${type}`}
+        aria-invalid
+        aria-live="assertive"
+        role="alert"
```

Comment:
Instead of adding these two attributes for each `<li>`, you can simply add them onto the surrounding `<ul>`. This will automatically announce the messages as they get rendered inside the `<ul>`.

Reply:
I've tested and if these two attributes are setted to `li` than screen reader automatically read the error message when it appears, and when attributes are setted to `ul` - screen reader does not read it automatically, only if hover to error message. I will leave attributes in `li` elements. Do you agree?

Reply:
Hmm. Interesting. It works in the Mac screen reader. Which screen reader are you using?

Reply:
I test with NVDA on Windows. I already checked again, the same behavior. Attributes should be in `li`

Reply:
I just checked out the various implementations of the `<AuthErrorsContainer>` and it's only rendered when there are errors. So that's why it's not working when the attributes are applied to the `<ul>` because it needs to stay in the DOM and then once it's children change (`<li>`), it will read out the updates.

But if it works on the `<li>`'s in both NVDA and JAWS then I think it's fine to leave them on each `<li>`.

However, I would like hear @JohnC-80's opinion on the matter too. I will approve it for now.